website/versioned_docs/version-29.1/WatchPlugins.md

```diff
@@ -130,7 +130,11 @@ Watch Usage
  › Press Enter to trigger a test run.
 ```
 
-**Note**: If the key for your plugin already exists as a default key, your plugin will override that key.
+:::note
```

Comment:
Thanks. Almost there – second note is missing in v29.1. Please add it. The rest is perfect.

Reply:
Just added it!